### PR TITLE
[docs] Http Authentication

### PR DESCRIPTION
**Pull request by churkin**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

**Comment by churkin** [comment]
ping @DevExpress/testcafe-docs

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 1]
```diff
@@ -64,6 +65,44 @@ If the start page is not specified, it defaults to `about:blank`.
...
+`credentials` | Object | Object that includes all necessary data for the authentication.
+
+Credentials parameter has following format:
+
+Parameter     | Type   | Description
+------------- | ------ | --------------------------------------------------------------------
+`username`    | String | Username for the authentication.
+`password`    | String | Password for the authentication.
+`workstation` | String | PC's ID in the local network (optional parameter).
+`domain`      | String | Domain name (optional parameter).
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/test-api/test-code-structure.md]: in other places we describe optional parameters in the following way:

`workstation` _(optional)_ | String | PC's ID in the local network.
`domain` _(optional)_     | String | Domain name.

**Comment by churkin** [comment]
FPR

**Comment by churkin** [comment]
ping @DevExpress/testcafe-docs

File: docs/articles/documentation/test-api/test-code-structure.md [hunk 2]
```diff
@@ -64,6 +65,35 @@ If the start page is not specified, it defaults to `about:blank`.
 > You can also call the [Navigate action](actions/navigate.md) at the beginning of a test
 > to navigate to the start page.
 
+### Http Authentication
+
+TestCafe allows you to test web pages that are protected with Http Basic or Windows (NTLM) authentication. To set authentication credentials use the `httpAuth` fixture declaration method.
```
Comment (inikulin) [inline docs/articles/documentation/test-api/test-code-structure.md]: Give links to wikipedia articles about authentication methods

**Comment by churkin** [comment]
FPR

**Comment by churkin** [comment]
ping /cc @DevExpress/testcafe-docs

**Comment by churkin** [comment]
@inikulin ping

**Comment by churkin** [comment]
FPR

**Comment by churkin** [comment]
FPR

**Pull request merged**